index.html

```diff
@@ -4,18 +4,44 @@
     <meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<link href="normalize.css" type="text/css" rel="stylesheet" />
-    <!-- Add character set when decided -->
 	<link href="style.css" type="text/css" rel="stylesheet" />
     <script src='index.js' defer></script>
 	<title>Quotlefish - A movie game</title>
 </head>
-
 <body>
-
+	<header>
+		<div>
+			<h1>Quotlefish</h1>
+		</div>
+		<div>
+			<img src="assets/info.png" id="buttonModal"/>
+		</div>
+	</header>
+	<main>
+		<section>
+			<div class="quoteContainer">
+				<p id="quote"></p>
+			</div>
+			<div id="modal">
```

Comment:
Not sure if the modal information should be in main. Think the contents of the modal button should be with the button (line 18).